### DISCUSSION TITLE
[Help]: How to disable slide change on swipe gesture?

### DISCUSSION THREAD

**Discussion by Miloszzz2** [davidjerleke/embla-carousel#804] [Help] [answered]
### Summary

Hello, I need a help. How to disable slide changing on swipe gesture. I can't find appropriate info in docs

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [X] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
Hi @Miloszzz2,

Set `watchDrag: false`. See [docs](https://www.embla-carousel.com/api/options/#watchdrag).

**Reply by Miloszzz2** [reply, marked as answer]
Thank you, that works 👍
